contracts/pcv/PCVGuardian.sol

```diff
@@ -81,7 +82,7 @@ contract PCVGuardian is IPCVGuardian, CoreRef {
         uint256 amount,
         bool pauseAfter,
         bool depositAfter
-    ) external override isGovernorOrGuardianOrAdmin {
```

Comment:
Admin will no longer be able to call the `withdraw()` functions - need to grant whichever addresses had this the PCV_SAFE_MOVER_ROLE if appropriate

Reply:
The "admin" should not be used at all going forward. All new contract deployments should explicitly enumerate the roles. I think the above methods should also get new modifiers

Reply:
Done